LIS/lis-test
# Added CopyFileDifferentMTU testcase in NET tests

Test case title: SCP with different MTU size
Steps: 1. Add two VM's on Hyper-V
2. Add an external network to each guest VM
3. Change MTU on each VM network to 9000
4. Create a large file (~1GB) on VM1
5. Send the file from VM1 to VM2 via SCP
6. Verify if transfer was successful by comparing both md5sums
Expected result: File transfer should be successful
